From 3622c5d33a319b22dec07619dda216213c304390 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 15 Nov 2014 13:05:25 +0100
Subject: [PATCH] Update license blurb in dpkg-repack to match the one in
 debian/copyright

The latter states that the code is under GPL-2+, but the former does not
mention the "or later" clause.
---
 dpkg-repack | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dpkg-repack b/dpkg-repack
index eea4ddf..61ed6cc 100755
--- a/dpkg-repack
+++ b/dpkg-repack
@@ -6,7 +6,8 @@
 #
 #   This program is free software; you can redistribute it and/or modify
 #   it under the terms of the GNU General Public License as published by
-#   the Free Software Foundation.
+#   the Free Software Foundation; either version 2 of the License, or
+#   (at your option) any later version.
 #
 #   This program is distributed in the hope that it will be useful,
 #   but WITHOUT ANY WARRANTY; without even the implied warranty of
-- 
2.1.3

